components/brave_wallet_ui/stories/locale.ts

```diff
@@ -885,5 +885,11 @@ provideStrings({
   braveWalletNftPinningErrorTooltipReasonTwo: 'Internal IPFS node problems',
   braveWalletNftPinningErrorTooltipReasonThree: 'Not enough space on local node',
   braveWalletImportNftModalTitle: 'Import NFT',
-  braveWalletEditNftModalTitle: 'Edit NFT'
+  braveWalletEditNftModalTitle: 'Edit NFT',
+  
+  // NFT auto discovery modal
+  braveWalletEnableNftAutoDiscoveryModalHeader: 'Want your NFTs displayed automatically?',
+  braveWalletEnableNftAutoDiscoveryModalDescription: 'Brave Wallet can uses a third-party service to automatically display your NFTs. Brave will share your public address with $1SimpleHash$2 to provide this service. $3Learn more.$4',
```

Comment:
```suggestion
  braveWalletEnableNftAutoDiscoveryModalDescription: 'Brave Wallet can uses a third-party service to automatically display your NFTs. Brave will share your public addresses with $1SimpleHash$2 to provide this service. $3Learn more.$4',
```

modified the text here so that it's clear we're sharing multiple addresses

Reply:
+1